src/builder/components/question/highlight/survey-question-highlight.component.tsx

```diff
@@ -0,0 +1,41 @@
+import { Icon } from 'antd';
+import React, { SFC } from 'react';
+import { SurveyItemType } from '../../../../common/types/survey-item.type';
+
+/**
+ * Displays a colorful decoration on the survey question card with an
+ * appropriate icon, depending on the survey question type
+ */
+
+export interface SurveyQuestionHighlightProps {
+    type: SurveyItemType;
+}
+
+export const SurveyQuestionHighlight: SFC<SurveyQuestionHighlightProps> = ({
+    type,
+}: SurveyQuestionHighlightProps): JSX.Element => {
+    // https://ant.design/components/icon/
+    const icons = {
```

Comment:
These consts could be declared outside of the function scope

Reply:
I never got why one over the other. Is it for readibility/reuse potential? Or am I missing something?

Reply:
https://github.com/decrn/localsurvey/pull/63/commits/dacec76ec71515def7e1b615b44bf362a7f137a2